packages/story-editor/src/components/form/stackable/stackableGroup.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import StackableInput from './stackableInput';
+
+/**
+ * Internal dependencies
+ */
+
+const StackableContainer = styled.div`
+  display: flex;
+  max-width: ${({ locked }) => (locked ? '106px' : 'none')};
+
+  & > ${StackableInput} {
+    border-radius: 0;
+
+    &:first-of-type > div {
+      border-radius: 4px 0 0 4px;
+      margin-left: 0;
+    }
+
+    &:last-of-type > div {
+      border-radius: 0 4px 4px 0;
+    }
```

Comment:
This fails if there's only one element because it would be both first of type and last of type and you overwrite the left radii in the declaration for the latter statement:

![Screen Shot 2021-11-05 at 11 52 16](https://user-images.githubusercontent.com/637548/140539681-6bae173b-4b26-4594-8399-428591ada40b.png)

You could add a third rule capturing both, but if you instead changed it to the following, it would be a bit prettier (and also more specific):
```css
    &:first-of-type > div {
      border-top-left-radius: 4px;
      border-bottom-left-radius: 4px;
      margin-left: 0;
    }

    &:last-of-type > div {
      border-top-right-radius: 4px;
      border-bottom-right-radius: 4px;
    }
```